doc/source/howtos/encrypt-traffic.rst

```diff
@@ -236,4 +236,46 @@ assumig there are two nodes in the cluster:
 
       wsrep_provider_options=socket.ssl=yes;socket.ssl_ca=/path/to/upgrade-ca.pem;socket.ssl_cert=/path/to/new-cert.pem;socket.ssl_key=/path/to/new-key.pem
 
+.. _ssl-auto-conf:
+
+SSL Automatic Configuration
+===========================
+
+|PXC| includes the :variable:`pxc-encrypt-cluster-traffic` variable
+that enables automatic configuration of SSL encrytion.
+By default, it is disabled, meaning that you need to configure SSL manually
+if you want to encrypt SST and IST traffic, and internal communication.
+
+This variable cannot be changed on the command line during runtime.
+To enable automatic configuration of SSL encryption,
+set ``pxc-encrypt-cluster-traffic=ON`` in the :file:`my.cnf` file.
+This will override encryption settings, but not other options related to SST,
+such as compression.
+
+When you enable automatic configuration of SSL encryption,
```

Comment:
This section is confusing.
@kennt-percona can you suggest better rewording.

Message has to clearly come

* If user set pxc-..=ON then user don't need to configure anything for SST/IST traffic
* if user set pxc-=ON and also specific SST/IST configuration what happens ?

@azhebel this needs to organized such that it comes up clearly for end-user.